### PR TITLE
Feature SERVO_TILT removed

### LINKED ISSUE #3009: Deprecat feature SERVO_GIMBAL
Gimbal output can be performed via mixer without any additional featrures.
- [ ] Drop servo_gimbal feature
- [ ] Add gimbal type options to mixer inputs

### PR DESCRIPTION
**Pull request by DzikuVx**
Closes #3009 

SERVO_TILT can be performed using only Mixer, without any additional features. Docs update will follow

### CODE REVIEW

File: src/main/flight/servos.c [hunk 1]
```diff
@@ -265,8 +252,21 @@ void servoMixer(float dT)
 
     input[INPUT_FEATURE_FLAPS] = FLIGHT_MODE(FLAPERON) ? servoConfig()->flaperon_throw_offset : 0;
 
-    input[INPUT_GIMBAL_PITCH] = scaleRange(attitude.values.pitch, -1800, 1800, -500, +500);
-    input[INPUT_GIMBAL_ROLL] = scaleRange(attitude.values.roll, -1800, 1800, -500, +500);
+    if (IS_RC_MODE_ACTIVE(BOXCAMSTAB)) {
+        const int gimbalPitch = scaleRange(attitude.values.pitch, -1800, 1800, -360, +360);
+        const int gimbalRoll = scaleRange(attitude.values.roll, -1800, 1800, -360, +360);
+
+        if (gimbalConfig()->mode == GIMBAL_MODE_MIXTILT) {
```
Comment (digitalentity) [inline src/main/flight/servos.c]: I'm not sure when MIXTILT is usable 🤔

**Comment by DzikuVx** [comment]
There are some very legacy servo gimbals that require this. Never seen one 😊

**Comment by digitalentity** [comment]
@DzikuVx maybe it's time we deprecate this MIXTILT thing?

**Comment by DzikuVx** [comment]
I think you are right. And if any really really really wants to have such, it is possible to mix gimbal roll and pitch using mixer itself. Will clean this up tomorrow

**Comment by DzikuVx** [comment]
And MIXTILT is gone. Looks good, Configurator changes will follow

**Review by digitalentity — APPROVED** [review]

**Pull request merged**